data/korath/korath jobs.txt

```diff
@@ -243,3 +243,593 @@ mission "Kor Efreti to Wanderers: Farming"
+	clearance "You read the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "fish trade" 3 10 0.8
+	source "La'ab Bamen"
+	stopover
+		government "Republic" "Pirate" "Free Worlds" "Neutral" "Independent"
+		attributes fishing
+	to offer
+		random < 40
+	on stopover
+		dialog "A local contact takes the trade goods from <origin> and gives you <tons> of fish."
+	on complete
+		payment 30000 180
+		dialog "A woman in a blue dress examines the fish and confirms the trade. You receive <payment>."
+		"reputation: Korath" += 2
+
+mission "Korath: Retrieve fish [2]"
+	name "Fish from <stopovers>"
+	job
+	repeat
+	description "Take <tons> of trade goods to <stopovers> to trade for <tons> of indigenous fish and return to <planet> for <payment>."
+	clearance "You read the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "fish trade" 7 25 0.8
+	source "La'ab Bamen"
+	stopover
+		government "Republic" "Pirate" "Free Worlds" "Neutral" "Independent"
+		attributes fishing
+	to offer
+		random < 40
+		"cargo space" > 10
+	on stopover
+		dialog "A local contact takes the trade goods from <origin> and gives you <tons> of fish."
+	on complete
+		payment 30000 180
+		dialog "A woman in a blue dress examines the fish and confirms the trade. You receive <payment>."
+		"reputation: Korath" += 3
+
+mission "Korath: Retrieve fish [3]"
+	name "Fish from <stopovers>"
+	job
+	repeat
+	description "Take <tons> of trade goods to <stopovers> to trade for <tons> of unusual fish and return to <planet> for <payment>."
+	clearance "You read the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "fish trade" 16 80 0.8
+	source "La'ab Bamen"
+	stopover
+		government "Republic" "Pirate" "Free Worlds" "Neutral" "Independent"
+		attributes fishing
+	to offer
+		random < 40
+		"cargo space" > 20
+	on stopover
+		dialog "A local contact takes the trade goods from <origin> and gives you <tons> of fish."
+	on complete
+		payment 30000 180
+		dialog "A woman in a blue dress examines the fish and confirms the trade. You receive <payment>."
+		"reputation: Korath" += 4
+
+mission "Korath: Retrieve fish [4]"
+	name "Fish from <stopovers>"
+	job
+	repeat
+	description "Take <tons> of trade goods to <stopovers> to trade for <tons> of unusual fish and return to <planet> for <payment>."
+	clearance "You read the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "fish trade" 35 300 0.8
+	source "La'ab Bamen"
+	stopover
+		government "Republic" "Pirate" "Free Worlds" "Neutral" "Independent"
+		attributes fishing
+	to offer
+		random < 40
+		"cargo space" > 50
+	on stopover
+		dialog "A local contact takes the trade goods from <origin> and gives you <tons> of fish."
+	on complete
+		payment 30000 180
+		dialog "A woman in a blue dress examines the fish and confirms the trade. You receive <payment>."
```

Comment:
Is the blue dress a uniform? Or just a popular color? Or are we always meeting the same lady who likes the same wearing the same outfit?

Reply:
Iris wore a white dress. I'll adjust the dialog so it does not describe the clothing; describing the clothing needlessly distracts from the rest of the information.

Reply:
It is now an "inventory officer" instead of a "woman in a blue dress."